root/etc/e-smith/events/actions/nethserver-dc-user-modify

```diff
@@ -22,10 +22,19 @@
 
 use strict;
 use Errno;
+use NethServer::SSSD;
 
 my $event = shift;
 my $userName = shift;
 my $FullName = shift;
+my $shell = shift || '/usr/libexec/openssh/sftp-server';
+
+my $sssd = new NethServer::SSSD();
+my $bindPassword = $sssd->bindPassword();
```

Comment:
The permissions associated to bindPassword() are going to change in the [near future](https://community.nethserver.org/t/special-account-for-applications-with-simple-ldap-bind/8175). We cannot assume write access with such credentials.

Reply:
which authentication to use @DavidePrincipi, we still need to authenticate I presume ?
this is what I have when I have no authentication
 `/usr/bin/systemd-run -M nsdc -q -t /usr/bin/ldbmodify -H ldap://localhost /tmp/stephane.ldif`
`ERR: (Operations error) "LDAP error 1 LDAP_OPERATIONS_ERROR -  <00002020: Operation unavailable without authentication> <>" on DN cn=stephane,cn=Users,dc=ad,dc=plop,dc=org at block before line 4
Modify failed after processing 0 records`